Formula/bubblewrap.rb

```diff
@@ -0,0 +1,30 @@
+class Bubblewrap < Formula
+  desc "Unprivileged sandboxing tool"
+  homepage "https://github.com/containers/bubblewrap"
+  url "https://github.com/containers/bubblewrap/releases/download/v0.4.1/bubblewrap-0.4.1.tar.xz"
+  sha256 "b9c69b9b1c61a608f34325c8e1a495229bacf6e4a07cbb0c80cf7a814d7ccc03"
+  license "LGPL-2.0-or-later"
+  depends_on "libcap"
+
+  def install
+    system "./configure", *std_configure_args, "--disable-silent-rules",
+      "--with-bash-completion-dir=#{share}/bash-completion/completions"
```

Comment:
Why not
```suggestion
          "--with-bash-completion-dir=#{bash_completion}"
```
? (additional spacing to align the arguments to `system`)

Reply:
Thanks! I amended the commit with this chance, and force-pushed it. Ready for review.

Reply:
I think you pushed the wrong commit.

Reply:
Yes! Sorry. Pushed it as normal commit now, it's shown in the merge request now.

I guess on merge they will be squashed together, so I thing it's fixed now.